dataService.js

```diff
@@ -0,0 +1,53 @@
+let categoryMap = null
+let productMap = null
+let combinedProductMap = null
+
+/*
+ * Map individual product objects to product id attributes
+ */
+function getProducts () {
+  if (!productMap || !productMap.size) {
+    const allProducts = require('./src/data/products.json').products
```

Comment:
Running `console.log(allProducts)` outside class throws `ReferenceError: allProducts is not defined` so there is no need to make it explicitly private